src/mame/bmc/koftball.cpp

```diff
@@ -207,7 +166,31 @@ void koftball_state::video_start()
 	m_tilemap[0]->set_transparent_pen(0);
 	m_tilemap[2]->set_transparent_pen(0);
 
-	m_pixbitmap = std::make_unique<bitmap_ind16>(0x400, 0x200);
+	m_tilemap[1]->set_transparent_pen(0);
+	m_tilemap[3]->set_transparent_pen(0);
+
+	m_pixbitmap = std::make_unique<bitmap_ind16>(512, 256);
+}
+
+// linear 512x256x4bpp
+void koftball_state::draw_pixlayer(bitmap_ind16 &bitmap, const rectangle &cliprect)
+{
+	const u8 pix_bank = (m_pixpal & 0xf) << 4;
+
+	for (int y = cliprect.min_y; y <= cliprect.max_y; y++)
+	{
+		const u16 pitch = y * 0x80;
+		for (int x = cliprect.min_x; x <= cliprect.max_x >> 2; x++)
+		{
+			const u16 tile_data = m_pixram[(pitch + x) & 0xffff];
+			for (int xi = 0; xi < 4; xi ++)
+			{
+				const u8 nibble = (tile_data >> ((3 - xi) * 4)) & 0xf;
+				if (nibble)
+					bitmap.pix(y, (x << 2) + xi) = pix_bank | nibble;
```

Comment:
Note: needs this switch because `copyscrollbitmap_trans` on its own is unable to break on a pal base. It's likely doable with `prio*` / `primask*` methods but I've forgot/don't know the incantation off the bat ...